src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource.cs

```diff
@@ -304,10 +304,9 @@ private SuggestedActionSet InlineActions(SuggestedActionSet actionSet)
                     // If the light bulb is only asking for code fixes, then we don't include suppressions.
                     var includeSuppressionFixes = requestedActionCategories.Contains(PredefinedSuggestedActionCategoryNames.Any);
 
-                    var fixes = Task.Run(
-                        () => _owner._codeFixService.GetFixesAsync(
-                                document, range.Span.ToTextSpan(), includeSuppressionFixes, cancellationToken),
-                        cancellationToken).WaitAndGetResult(cancellationToken);
+                    var fixes = ThreadingContext.JoinableTaskFactory.Run(
+                        async () => await _owner._codeFixService.GetFixesAsync(
+                                document, range.Span.ToTextSpan(), includeSuppressionFixes, cancellationToken));
```

Comment:
why was this change made?

Reply:
I was asked to make it possible to use UI thread for both refactoring and codefix.